onnx/backend/test/case/node/sequenceinsert.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+def sequence_insert_reference_implementation(sequence, tensor, position=None):  # type: (np.ndarray, np.ndarray, np.ndarray) -> np.ndarray
+    if position:
+        # In these cases, insert_position will be between [-len(sequence), len(sequence)]
+        # The position argument will be in the format np.array([pos_index])
+        insert_position = position[0]
+        sequence.insert(insert_position, tensor)
+    else:
+        # Default position of insertion is at the end of the sequence.
+        sequence.append(tensor)
+    return np.array(sequence)
+
+
+class SequenceInsert(Base):
+
+    @staticmethod
+    def export():  # type: () -> None
+        test_cases = {
+            'at_back': [np.array([10, 11, 12])],
+            'at_front': [np.array([-2, -1, 0]), np.array([0])]
+        }
+        sequence = [np.array([1, 2, 3, 4]), np.array([5, 6, 7]), np.array([8, 9])]
+
+        for test_name, test_inputs in test_cases.items():
+            tensor = test_inputs[0]
+
+            if len(test_inputs) > 1:
+                node = onnx.helper.make_node(
+                    'SequenceInsert',
+                    inputs=['sequence', 'tensor', 'position'],
+                    outputs=['output_sequence']
+                )
+                position = test_inputs[1]
+                inserted = sequence_insert_reference_implementation(sequence, tensor, position)
```

Comment:
see above comment, i think that's probably the reason why the generated test data file is not expected. The `sequence` at this point is already updated by the other branch.